bls.go

```diff
@@ -56,41 +56,33 @@ func concatAppend(slices [][]byte) []byte {
 }
 
 // SerializeBig serializes a public key uncompressed.
-func (p PublicKey) SerializeBig() [193]byte {
+func (p PublicKey) SerializeBig() []byte {
```

Comment:
Is there any reason for this change? Uncompressed public keys are always going to be 193 bytes correct?